reference/orders.v2.oas2.yml

```diff
@@ -4075,11 +4063,11 @@ components:
           example: '0.0000'
           type: string
         base_wrapping_cost:
-          description: 'The value of the base wrapping cost expressed as a floating point number to four decimal places in string format.'
+          description: The value of the base wrapping cost expressed as a floating point number to four decimal places in string format.
           example: '0.0000'
           type: string
         billing_address:
-          $ref: '#/components/schemas/billingAddress_Full'
+          $ref: '#/components/schemas/billingAddress_Base'
```

Comment:
may you pls add (in the PR description) some stoplight studio screenshots of:
- the billing_address in POST/PUT request payload does not contain the `form_field` field
- the billing_address in GET response contains the `form_field` field as read-only

I doubt changing this `order_Shared` will remove the `form_field` from the GET response as well.

Reply:
@donald-nguyen-bc, I assisted @bc-tgomez with this, so I am chiming in:

`POST` `v2/orders` and  `PUT` `v2/orders/{order_id}`: 
![image](https://user-images.githubusercontent.com/29524703/194400348-6a391805-010f-472c-91e1-ae1404a11ce4.png)

`GET` `v2/orders`: 
![image](https://user-images.githubusercontent.com/29524703/194400611-beea9483-cf9b-472f-a69a-6332bfee2c69.png)

Reply:
awesome @markcmurphy, it looks good